invenio_communities/communities/services/service.py

```diff
@@ -20,16 +20,30 @@
 class CommunityService(RecordService):
     """community Service."""
 
-    def __init__(self, config, files_service=None):
+    def __init__(
+            self, config, files_service=None, invitations_service=None,
+            members_service=None):
         """Constructor for CommunityService."""
         super().__init__(config)
         self._files = files_service
+        self._invitations = invitations_service
+        self._members = members_service
```

Comment:
Placing them all under the communities service... The members and invitations services are pretty coupled. Maybe they are not super coupled to the communities one. The programmatic API is nicer looking this way :shrug:.

Reply:
Agreed. I am waiting to see if the abstraction will reveal itself as we go.